pkg/volume/fc/fc_util.go

```diff
@@ -60,7 +60,7 @@ func (handler *osIOHandler) WriteFile(filename string, data []byte, perm os.File
 
 // given a wwn and lun, find the device and associated devicemapper parent
 func findDisk(wwn, lun string, io ioHandler, deviceUtil volumeutil.DeviceUtil) (string, string) {
-	fcPath := "-fc-0x" + wwn + "-lun-" + lun
+	fcPath := "fc-0x" + wwn + "-lun-" + lun
```

Comment:
Can you please add a corresponding unit test in fc_utils_test.go

Reply:
@saad-ali done, added the UT for the new scenario, and for the missing negative case as well.